docs/internal/templates/specifications/tests/TestCaseIdentifierSpecification.md

```diff
@@ -1,7 +1,7 @@
 # Test Case Identifier Specification
 
-All test cases belong to one specific test level, and it gets its name from that
-test level's name. The following test levels are defined and available:
+All test cases belong to one module and their names are based on that module’s
+name. The following modules are defined and available:
```

Comment:
This document now refer "modules" but previously it referred to "test levels". We should use the same terminology in all places. The reason why this document referred to "test level" is because it is called so in other documents. We should be consistent.

See e.g. https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/MasterTestPlan.md#definition-of-test-levels

Reply:
I think the current wording across component is Module,

* Test zone data : https://github.com/zonemaster/zonemaster/blob/397c1281da08d0f3323fae6eae1ab54ec0779838/test-zone-data/README.md?plain=1#L94C1-L96C62
* Backend RPCAPI document : https://github.com/zonemaster/zonemaster/blob/397c1281da08d0f3323fae6eae1ab54ec0779838/docs/public/using/backend/rpcapi-reference.md?plain=1#L328
* CLI usage : https://github.com/zonemaster/zonemaster-cli/blob/master/lib/Zonemaster/CLI.pm#L119-L127

and so on.